### DISCUSSION TITLE
utools下极简发布memo

### DISCUSSION THREAD

**Discussion by linbuxiao** [flomoapp/3rd-party-tools#59] [分享创造]
- 大小仅有45k。
- `floapi` - 设置你的api地址
- `flo` - 输入后回车，即可发布flomo
- `github` - https://github.com/linbuxiao/utools-flomo
- 安装文件地址 - https://github.com/linbuxiao/utools-flomo/releases

![动画 (2)](https://user-images.githubusercontent.com/56839018/129157402-bd33a3a7-ecda-444b-8e28-118f34f4e1f6.gif)

**Comment by linbuxiao** [comment]
该插件会长期维护，如果有问题在这里提出，或者到项目提出issue也可~
email: linbuxiao@gmail.com

**Reply by kubulo** [reply]
怎么使用，没有说明呢，下载一个压缩包zip？

**Reply by linbuxiao** [reply]
> 怎么使用，没有说明呢，下载一个压缩包zip？

有打包的 upx 文件。这个格式是 utool 支持拖入安装的~
